### DISCUSSION TITLE
remove editor language drop down menu from project manager

### DISCUSSION THREAD

**Discussion by bossoverdose** [godotengine/godot-proposals#6259] [Editor]
set **editor language** exists in **editor settings**, is unnecessary the duplication, this is set only one time, makes no sense to have it all the time in the project manager.

just remove **editor language** drop down menu from project manager.

**Comment by Calinou** [comment]
The project manager allows you to change the language directly, so you don't have to open the editor and click buttons you don't understand the text of :slightly_smiling_face:
